extensions/ui/org.eclipse.smarthome.ui.paper/package.json

```diff
@@ -4,13 +4,15 @@
   "private": true,
   "dependencies": {
     "bower": "^1.4.1",
+    "browser-sync": "~2.7.12",
     "del": "^2.0.0",
     "gulp": "^3.9.0",
     "gulp-angular-filesort": "~1.1.1",
     "gulp-concat": "^2.6.0",
     "gulp-ng-annotate": "~1.0.0",
     "gulp-rename": "^1.2.2",
-    "gulp-uglify": "^1.5.1"
+    "gulp-uglify": "^1.5.1",
+     "http-proxy-middleware": "^0.9.0"
```

Comment:
That is correct. #847 was actually fixed with #873. If you look at https://github.com/eclipse/smarthome/blob/master/extensions/ui/org.eclipse.smarthome.ui.paper/build.properties the only files that are included are inside of the web folder. So at this point the only way things should end up in /web/ is through the `gulp build` command. 

To hopefully help you understand how the UI is currently structured. All files that are included are located at the top of the gulp file in the `paths` variable, https://github.com/eclipse/smarthome/blob/bd2bab036da3ed9daaabaaeffdcabed0ea26910b/extensions/ui/org.eclipse.smarthome.ui.paper/gulpfile.js#L11 with the exception of `index.html`. 

If we add future files to the `paths` variable it should make it very easy to see all of the files and resources that are being packaged within the jar. All external resources should come from `bower_components` or `node_modules`, with maybe a few exceptions down the road. As it currently stands everything is coming from bower. 

I will always inform you if a new resource is added or the version is updated and glad to answer your questions, especially after you answered a billion from me :smile: